test/fw/ptl/utils/pbs_procutils.py

```diff
@@ -281,24 +281,23 @@ def set_frequency(self, value=60):
         self.logger.debug('procmonitor: set frequency to ' + str(value))
         self.frequency = value
 
-    def get_system_stats(self, nw_protocols=sentinel):
+    def get_system_stats(self, nw_protocols=None):
         """
         Run system monitoring
         """
-        if nw_protocols is sentinel:
-            self.skipTest("network protocol values is not set.")
+        # if no protocols set, use default
+        if not nw_protocols:
+            nw_protocols = ['TCP']
         cmd = 'sar -rSub -n %s 1 1' % ','.join(nw_protocols)
-        try:
-            rv = self.du.run_cmd(cmd=cmd, as_script=True)
-        except BaseException:
-            msg = f"Is sar package installed"
-            self.logger.error(msg)
-            self.skipTest(msg)
+        rv = self.du.run_cmd(cmd=cmd, as_script=True)
+        if rv['err']:
+            return False
```

Comment:
@vstumpf self.du.run method itself check command available or not. it through error "command not available"
Now test doesn't check sar command related data.I have atteched the log when command not available.

Reply:
I think get_system_stats() should return the dictionary already made. That way, you won't have to create self.sysstat AND the _sys_info dictionaries. If the command errors, just return None.
PS: you can also make dictionaries in one statement, like so:
```Python3
_sys_info = {
    'name': "System",
    'time': time.ctime(timenow),
     ...
}
return _sys_info
```

Reply:
@vstumpf done the changes.